### ISSUE TITLE
Notification - Target Size WCAG rule 2.5.5

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7614]
**Failed rule**

2.5.5
Target Size (Enhanced)
Level AAA(Added in 2.1)

The size of the target for pointer inputs is at least 44 by 44 CSS pixels except when: 

- **Equivalent**: The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;
- **Inline**: The target is in a sentence or block of text;
- **User Agent Control**: The size of the target is determined by the user agent and is not modified by the author;
- **Essential**: A particular presentation of the target is essential to the information being conveyed.

---

**Notes**

This is a Level AAA fail.
